include/rocksdb/options.h

```diff
@@ -1228,6 +1228,9 @@ struct DBOptions {
   // file.
   bool manual_wal_flush = false;
 
+  // If enabled WAL records will be compressed before they are written.
```

Comment:
This needs to be marked EXPERIMENTAL or WORK IN PROGRESS

Reply:
Addressed.